app/assets/xml/search.xml.erb

```diff
@@ -0,0 +1,9 @@
+<% helper = Module.new { extend ApplicationHelper } %>
```

Comment:
A little hack to make the helpers available here so we can reuse `community_name` and `community_qualified_name` to make sure they stay in sync with the rest of the app.